src/Bundle/Resources/config/services/metadata/resource_metadata_collection.php

```diff
@@ -62,6 +63,15 @@
             service('.inner'),
         ]);
 
+    $services->set('sylius.resource_metadata_collection.factory.plural_name', PluralNameResourceMetadataCollectionFactory::class)
+        ->decorate('sylius.resource_metadata_collection.factory', null, 300)
```

Comment:
we can reuse the same priority as the next one. It doesn't matter.
and of course, if this is configured before, it will be first.